script/hassfest/dependencies.py

```diff
@@ -156,15 +157,19 @@ def visit_Attribute(self, node):
 
 def calc_allowed_references(integration: Integration) -> set[str]:
     """Return a set of allowed references."""
+    manifest = integration.manifest
+    if not manifest:  # No manifest, nothing allowed.
```

Comment:
Use a wallrus :=

Reply:
Addressed in https://github.com/home-assistant/core/pull/82319/files